docs/docs/customize/layout-engines.md

```diff
@@ -100,3 +100,9 @@ context.PluginManager.AddPlugin(treeLayoutPlugin);
 ```
 
 ![TreeLayoutEngine demo](../../images/tree-layout-demo.gif)
+
+### `FreeLayoutEngine`
+
+<xref:Whim.Layout.FreeEngineLayout> is a layout that allows users to have only free-floating windows.
```

Comment:
The `xref` should be `<xref:Whim.FreeLayoutEngine>`.

```suggestion
<xref:Whim.FreeLayoutEngine> is a layout that has all windows being free-floating. To have specific windows float within a different layout, see the [Floating Layout Plugin](../plugins/floating-layout.md).
```

Reply:
Done